esb-connector-automation/archetype-resources/pom.xml

```diff
@@ -0,0 +1,355 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+
+   WSO2 Inc. licenses this file to you under the Apache License,
+   Version 2.0 (the "License"); you may not use this file except
+   in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied. See the License for the
+   specific language governing permissions and limitations
+   under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.carbon.connector</groupId>
+    <artifactId>${artifactId}</artifactId>
+    <version>1.0.0</version>
+    <packaging>bundle</packaging>
+    <name>WSO2 Carbon - Connector For ${connector_name}</name>
+    <url>http://wso2.org</url>
+
+    <properties>
+        <axis2.osgi.version.range>1.7.1</axis2.osgi.version.range>
+        <axiom.osgi.version.range>1.2.5</axiom.osgi.version.range>
+        <connector.name>${connector_name}</connector.name>
+        <test.framework.version>4.2.0</test.framework.version>
```

Comment:
Yes, This is used as template to generate the connector source code. We need these versions for connector integration test since it's generate source code for test case also.